docs/application/dotnet/guides/nui/flex-layout.md

```diff
@@ -0,0 +1,117 @@
+| `Justification`        | FlexJustification | Alignment of items along the flex axis when free space is available |
+| `Alignment`            | AlignmentType     | Alignment of items along the cross axis when free space is available |
+| `WrapType`             | FlexWrapType    | Enable wrapping of items |
+
+Natural size of items are used, which can be different for each item. Additionally, setting the size of an item has no effect.
+
+`Justification` applies to the flex Direction axis whilst `Alignment` is the (other) cross axis. Changing the Direction will apply the `Justification` to the new direction.
+
+
+## FlexDirection
+
+`Direction` property specifies the main axis direction along which the flex items are placed.
+
+![FlexDirection](./media/flexDirection.png)
+
+| Property value |  Description |
+|----------------|--------------|
+| `Row`          | Place items horizontally in a row |
+| `RowReverse`   | Place items horizontally in a row, but in reverse order |
+| `Column`       | Place items vertically in a column |
+| `ColumnReverse` | Place items vertically in a column, but in reverse order |
+
+The following code shows how to set the `Direction` property using `FlexDirection`:
+
+```csharp
+View layoutView = new View();
+var flexLayout = new FlexLayout();
+flexLayout.Direction = FlexLayout.FlexDirection.Column;
+layoutView.Layout = flexLayout;
+```
+
+
+## FlexJustification
+
+`Justification` property specifies the alignment for flex items, when they do not use all available space on the main axis.
+
+![FlexJustification](./media/justifyContent.png)
+
+| Property value  |   Description |
+|-----------------|---------------|
+| `FlexStart`     | Position items at the beginning of the container |
+| `FlexEnd`       | Position items at the end of the container |
+| `Center`        | Position items at the center of the container |
+| `SpaceBetween`  |  Position items with equal spacing between them |
+| `SpaceAround`   |  Position items with equal spacing before, between, and after them |
+
+The following code shows how to set the `Justification` property using `FlexJustification`:
+
+```csharp
+View layoutView = new View();
+var flexLayout = new FlexLayout();
+flexLayout.Justification = FlexLayout.FlexJustification.SpaceBetween;
+layoutView.Layout = flexLayout;
+```
+
+
+## AlignmentType
+
+`Alignment` property specifies the alignment for flex items when they do not use all the available space on the cross axis.
+
+![AlignmentType](./media/alignItems.png)
+
+|  Property value  |   Description  |
+|------------------|----------------|
+|  `Auto`          | Inherit the same alignment from the parent |
+|  `FlexStart`     | Align items to the beginning of the container |
+|  `Center`        | Align items to the center of the container |
+|  `FlexEnd`       | Align items to the end of the container |
+|  `Stretch`       | Stretch items to fit the container |
+
```

Comment:
Just a suggestion. How about applying the same order to the items in the table with the image?
FlexStar, FlexEnd, Center, and Stretch